scan/git.go

```diff
@@ -306,3 +307,10 @@ func git(args ...string) ([]byte, error) {
 func isGitTransport(str string) bool {
 	return urlutil.IsURL(str) || strings.HasPrefix(str, "git://") || strings.HasPrefix(str, "git@")
 }
+
```

Comment:
I think we also need to handle `visualstudio.com` and `dev.azure.com` for Azure DevOps

Reply:
Removed 'github' from the pattern.